include/extractor/extraction_way.hpp

```diff
@@ -44,9 +47,16 @@ struct ExtractionWay
     void set_backward_mode(const TravelMode m) { backward_travel_mode = m; }
     TravelMode get_backward_mode() const { return backward_travel_mode; }
 
+    // speed in km/h
     double forward_speed;
     double backward_speed;
+    // weight per meter
+    double forward_weight_per_meter;
+    double backward_weight_per_meter;
```

Comment:
are these per meter, or per decimeter as indicated by the division by ten in assembly?